### PR TITLE
N17: rg3d

### PR DESCRIPTION
**Pull request by mrDIMAS**
For #408

### CODE REVIEW

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -276,6 +276,39 @@ _Discussions:
 [rayon]: https://github.com/rayon-rs/rayon
 [crates.io]: https://crates.io/
 
+### [rg3d]
+
+[![rusty-editor screenshot](rusty-editor.jpg)][rg3d_twit]
+_A level made in [rusty-editor][rusty_editor] which a native scene editor for rg3d game engine._
```
Comment (Uriopass) [inline content/posts/newsletter-017/index.md:282]: Please use less than 80 characters per line

File: content/posts/newsletter-017/index.md [hunk 2]
```diff
@@ -276,6 +276,39 @@ _Discussions:
...
+- Added verbosity levels for logger
+- Improved lightmap quality
+- Added proper syncing between physics and scene graph
+- Improved examples
+- Added cylinder emitter for particle systems
+- Improved lots of UI widgets and added VectorImage widget
+- rg3d-sound is now able to manage multiple contexts
+- [hrtf][hrtf] crate now has 2 times better performance
+- Added support for 64-bit FBX format (version 7500+)
+- rusty-editor now has [navigation meshes][rusty_editor_navmesh] and particle systems editor
```
Comment (Uriopass) [inline content/posts/newsletter-017/index.md:297]: Here too

**Review by Uriopass — CHANGES_REQUESTED** [review]

**Review by ozkriff — APPROVED** [review]
:fire:

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**